ethcore/builtin/src/lib.rs

```diff
@@ -215,61 +217,58 @@ impl Builtin {
 	}
 }
 
-impl From<ethjson::spec::Builtin> for Builtin {
-	fn from(b: ethjson::spec::Builtin) -> Self {
+impl TryFrom<ethjson::spec::Builtin> for Builtin {
+	type Error = Error;
```

Comment:
nitpick: I think this would have been more readable: `type Error = EthcoreError`.

Reply:
Good point